### PR TITLE
Put tags and categories to blog pages

### LINKED ISSUE #4958: Put tags/categories to Blog pages
**What would you like to be added**:

Tag or/and categorize Blog pages so that users can easily search.

**Why is this needed**:

Currently, https://pipecd.dev/blog/ contains some kinds of articles (e.g. announcements, best practices)
Users cannot easily find the pages they're looking for.

### PR DESCRIPTION
**Pull request by t-kikuc**
**What this PR does / why we need it**:

as title.

**Which issue(s) this PR fixes**:

Fixes #4958

**Does this PR introduce a user-facing change?**:　N/A

- **How are users affected by this change**:N/A
- **Is this breaking change**:N/A
- **How to migrate (if breaking change)**:N/A

**Example**

each blog page:

image

search by category (click any category):

image

search by tag (click any tag):

image

### CODE REVIEW

**Review by Warashi — DISMISSED** [review]
Thank you!

File: docs/content/en/blog/may-2022-update.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ linkTitle: "May 2022 update"
 weight: 993
 description: "Development status update to recap what happened last few months"
 author: Khanh Tran ([@khanhtc1202](https://twitter.com/khanhtc1202))
+categories: ["Announcement"]
+tags: ["What's new"]
```
Comment (khanhtc1202) [inline docs/content/en/blog/may-2022-update.md:9]:
```suggestion
tags: ["News"]
```
nits

File: docs/content/en/blog/pipecd-at-kubecon-eu-2024.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ title: "PipeCD at CNCF KubeCon EU 2024"
 linkTitle: "PipeCD at CNCF KubeCon EU 2024"
 weight: 988
 author: Khanh Tran ([@khanhtc1202](https://github.com/khanhtc1202))
+categories: ["Announcement"]
+tags: ["Conference"]
```
Comment (khanhtc1202) [inline docs/content/en/blog/pipecd-at-kubecon-eu-2024.md:8]:
```suggestion
tags: ["Conference", "News"]
```
nits

File: docs/content/en/blog/play-environment.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ linkTitle: "The PipeCD play environment"
 weight: 994
 description: "In this post, we will have a glance at the play environment of PipeCD, how to access and what you can get from the environment."
 author: Khanh Tran ([@khanhtc1202](https://twitter.com/khanhtc1202))
+categories: ["Announcement"]
+tags: ["Getting Started"]
```
Comment (khanhtc1202) [inline docs/content/en/blog/play-environment.md:9]:
```suggestion
tags: ["Getting Started", "News"]
```

**Comment by t-kikuc** [comment]
@khanhtc1202 Thanks! I fixed them all! plz check again 🙏

**Review by khanhtc1202 — APPROVED** [review]
LNeatTM 👍

**Review by Warashi — APPROVED** [review]
LGTM

**Comment by t-kikuc** [comment]
I opened an issue https://github.com/pipe-cd/pipecd/issues/4965 because the tags and the categories do not appear.
Maybe that's due to the version of Hugo.

**Pull request merged**